### ISSUE TITLE
Version should be controlled based upon plugin-name, so that versions don't conflict between plugins

### ISSUE THREAD

**Issue by Ferrariic** [Bot-detector/bot-detector#163]
![image](https://user-images.githubusercontent.com/5789682/187114238-d73df4e1-3328-4306-8f8d-67a99702c128.png)

**Comment by Ferrariic** [comment]
Atm the NeverScapeAlone plugin also uses `version` in the text file (this will be changed in the future), but BDP also uses the same versioning system in the plugin properties

Current version
```
task createProperties(dependsOn: processResources) {
	doLast {
		new File("$buildDir/resources/main/version.txt").text = "version=$project.version"
	}
}
```

maybe we should change this to:

```
task createProperties(dependsOn: processResources) {
	doLast {
		new File("$buildDir/resources/main/botdetectorversion.txt").text = "version=$project.version"
	}
}
```

**Comment by Cyborger1** [comment]
Will be fixed with https://github.com/runelite/plugin-hub/pull/3235

**Comment by Cyborger1** [comment]
Fixed forever in theory by #164

**Comment by Cyborger1** [comment]
This should no longer be an issue
